src/server/service/debug.rs

```diff
@@ -21,6 +21,7 @@ fn error_to_status(e: Error) -> RpcStatus {
         Error::NotFound(msg) => (RpcStatusCode::NOT_FOUND, msg),
         Error::InvalidArgument(msg) => (RpcStatusCode::INVALID_ARGUMENT, msg),
         Error::Other(e) => (RpcStatusCode::UNKNOWN, format!("{:?}", e)),
+        Error::EngineTrait(e) => (RpcStatusCode::NOT_FOUND, format!("{:?}", e)),
```

Comment:
```suggestion
        Error::EngineTrait(e) => (RpcStatusCode::UNKNOWN, format!("{:?}", e)),
```

I think it should be `UNKNOWN`, there are many other errors in `Error::EngineTrait`.